php/Blockonomics.php

```diff
@@ -401,50 +401,64 @@ public function is_lite_mode_active(){
     }
 
     // Adds the header to the blockonomics page
-    public function load_blockonomics_header($template_name){
-        add_action('wp_enqueue_scripts', 'bnomics_enqueue_stylesheets' );
-        // Don't load javascript files if no js is active
-        if (!$this->is_nojs_template($template_name)) {
-            add_action('wp_enqueue_scripts', 'bnomics_enqueue_scripts' );
-        }
+    public function load_blockonomics_header($template_name, $additional_script=NULL){
         // Lite mode will render without wordpress theme headers
         if($this->is_lite_mode_active()){
         ?>
-          <link rel="stylesheet" type="text/css" href="<?php echo plugins_url('css/order.css', dirname(__FILE__));?>">
+            <link rel="stylesheet" type="text/css" href="<?php echo plugins_url('css/order.css', dirname(__FILE__));?>">
+            <script src="<?php echo plugins_url('js/vendors/reconnecting-websocket.min.js', dirname(__FILE__));?>" defer="defer"></script>
```

Comment:
Both light mode and nojs can be activated at the same time. We should still be checking `if (!$this->is_nojs_template($template_name))` when light mode is active. Currently the JS is loaded when both lite mode and nojs settings are activated.